pkg/plugins/policies/meshpassthrough/plugin/xds/listeners_filter_chain.go

```diff
@@ -13,6 +14,8 @@ import (
 	xds_virtual_hosts "github.com/kumahq/kuma/pkg/xds/envoy/virtualhosts"
 )
 
+const NoMatchMsg = "This is a Kuma Sidecar. No routes match this Domain!\n"
```

Comment:
I'm not sure about this message. Maybe @lahabana or @jakubdyszkiewicz could give some input here.

Reply:
Yeah, the wording here needs to be improved.